datasets/brk/kadastraleobjecten/v1.1.1.json

```diff
@@ -241,6 +241,7 @@
         "description": "Dit veld is all\u00e9\u00e9n gevuld wanneer het beschreven kadastrale object een A-perceel betreft."
       },
       "heeftEenRelatieMetVerblijfsobject": {
+        "shortname": "hftRelMtVot",
```

Comment:
@jjmurre geen andere shortname? `heeftRelMetVerbl` ofzo?

Reply:
Er is een officiele systematiek voor de afkortingen van de BenK gerelateerde namen: https://wiki.data.amsterdam.nl/doku.php?id=start:datasets:basisregistratie:afkortingen_registraties
Verblijfsobject wordt dan bijv. Vot (zie er vreemd uit, maar het went).